README.md

```diff
@@ -140,7 +144,7 @@ spec:
 
 ### 4. Ensuring the Certificate request has been fulfiled
 
-Currently, cert-manager does not log Events on Certificates or Issuers to the
+> Currently, cert-manager does not log Events on Certificates or Issuers to the
```

Comment:
❓ This gets rendered as quoted text. Is that intended?

Reply:
Yep it was